_posts/meat/2020-06-07-pork-larb.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Pork Larb
+category: meat
+tags: recipeish
+---
+
+Writing this up so that Buro can make it too. <3
+
+Ingredients
+
+* 1 pound ground pork
```

Comment:
These ingredients can be grouped a bit. Like: 

```
For serving: 
* 1/2 cup - 1 cup white jasmine/sticky rice, or
* lettuce / chard leaves for serving (optional)

Garnishes (optional): 
* 1/2 bunch green onions, chopped (optional)
* 1/2 bunch cilantro, chopped (optional)
* few sprigs mint, chopped (optional) 

```

Reply:
the rice isn't for serving tho it's for toasting

Reply:
Oh derp